### PR TITLE
Navbars

### PR DESCRIPTION
**Pull request by samandar0806**

### CODE REVIEW

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                      d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM9 15a1 1 0 011-1h6a1 1 0 110 2h-6a1 1 0 01-1-1z"
+                      clip-rule="evenodd"
+                    ></path>
+                  </svg>
+                </button>
+              </div>
+              <nav className="flex-col flex-grow hidden pb-4 md:pb-0 md:flex md:justify-end md:flex-row">
+                <a
+                  className="px-4 py-2 mt-2 text-sm font-semibold bg-transparent rounded-lg dark-mode:bg-transparent dark-mode:hover:bg-gray-600 dark-mode:focus:bg-gray-600 dark-mode:focus:text-white dark-mode:hover:text-white dark-mode:text-gray-200 md:mt-0 md:ml-4 text-white hover:duration-500 hover:text-[#F16262] focus:outline-none"
+                  href="#"
```
Comment (ibrohim5009) [inline src/components/navbar/Navbar.jsx]: whot bu uqib bomittiku

**Review by ibrohim5009 — CHANGES_REQUESTED** [review]

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                      fill-rule="evenodd"
+                      d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM9 15a1 1 0 011-1h6a1 1 0 110 2h-6a1 1 0 01-1-1z"
+                      clip-rule="evenodd"
+                    ></path>
+                  </svg>
+                </button>
+              </div>
+              <nav className="flex-col flex-grow hidden pb-4 md:pb-0 md:flex md:justify-end md:flex-row">
+                <a
+                  className="px-4 py-2 mt-2 text-sm font-semibold bg-transparent rounded-lg dark-mode:bg-transparent dark-mode:hover:bg-gray-600 dark-mode:focus:bg-gray-600 dark-mode:focus:text-white dark-mode:hover:text-white dark-mode:text-gray-200 md:mt-0 md:ml-4 text-white hover:duration-500 hover:text-[#F16262] focus:outline-none"
```
Comment (xakimovotabek200) [inline src/components/navbar/Navbar.jsx:33]: image
mediayada xato bor eka

File: src/components/navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                  href="#"
+                >
+                  Blog
+                </a>
+                <a
+                  className="px-4 py-2 mt-2 text-sm font-semibold bg-transparent rounded-lg dark-mode:bg-transparent dark-mode:hover:bg-gray-600 dark-mode:focus:bg-gray-600 dark-mode:focus:text-white dark-mode:hover:text-white dark-mode:text-gray-200 md:mt-0 md:ml-4 text-white hover:duration-500 hover:text-[#F16262] "
+                  href="#"
+                >
+                  Portfolio
+                </a>
```
Comment (akbarali0705) [inline src/components/navbar/Navbar.jsx:43]: Qolgan hammasida onClick bo'lganda effect bor lekin bitta bunda yo'q

**Review by akbarali0705 — COMMENTED** [review]
Qolgan hammasida onClick bo'lganda effect bor lekin bitta bunda yo'q

File: src/components/navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+              </nav>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default Navbar;
```
Comment (tolibjon1015) [inline src/components/navbar/Navbar.jsx]: pretterni to'g'irlang

**Review by tolibjon1015 — CHANGES_REQUESTED** [review]

File: src/components/navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                  About
+                </a>
+                <a
+                  className="px-4 py-2 mt-2 text-sm font-semibold bg-transparent rounded-lg dark-mode:bg-transparent dark-mode:hover:bg-gray-600 dark-mode:focus:bg-gray-600 dark-mode:focus:text-white dark-mode:hover:text-white dark-mode:text-gray-200 md:mt-0 md:ml-4 text-white hover:text-[#F16262] hover:duration-500 focus:outline-none focus:shadow-outline"
+                  href="#"
+                >
+                  Contact
+                </a>
+                <button className="  hidden sm:block sm: relative items-center justify-center p-0.5 mb-2 mr-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-purple-600 to-blue-500 group-hover:from-purple-600 group-hover:to-blue-500 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800">
+                  <span className="relative px-5 py-2.5 transition-all ease-in duration-75 bg-white dark:bg-gray-900 rounded-md group-hover:bg-opacity-0">
```
Comment (Muhammadali2oo6) [inline src/components/navbar/Navbar.jsx]: Buttonda kamchilik bor!!!!!!

**Review by Muhammadali2oo6 — APPROVED** [review]

File: src/components/navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+      ></script>
+      <div className="min-h-screen ">
+        <div className="antialiased bg-gray-200 dark-mode:bg-gray-900">
+          <div className=" text-gray-700 bg-white dark-mode:text-gray-200 dark-mode:bg-gray-800 ">
+            <div
+              x-data="{ open: true }"
+              className="flex flex-col px-40 mx-auto md:items-center md:justify-between md:flex-row  bg-[#292937]"
+            >
+              <div className="flex items-center justify-between p-4 ">
+                <img src={Logo} alt="" />
```
Comment (xakimovotabek200) [inline src/components/navbar/Navbar.jsx:15]: Hover bolganda cursor-pointer bolish kerak

**Review by xakimovotabek200 — CHANGES_REQUESTED** [review]

File: src/components/navbar/Navbar.jsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import Logo from "../../images/logo-white.png";
+
+function Navbar() {
+  return (
+    <div>
+      <script
+        src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.x.x/dist/alpine.min.js"
+        defer
+      ></script>
```
Comment (tolibjon1015) [inline src/components/navbar/Navbar.jsx]: script nima qilyapti ????????????????????????????????????????????????????????????????????

File: src/components/navbar/Navbar.jsx (hunk 4 above)
Comment (Ahmadullo21) [inline src/components/navbar/Navbar.jsx]: pritrni ??????

**Review by Ahmadullo21 — APPROVED** [review]

File: src/components/navbar/Navbar.jsx [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+      <div class="min-h-screen ">
+        <div class="antialiased bg-gray-100 dark-mode:bg-gray-900">
+          <div class=" text-gray-700 bg-white dark-mode:text-gray-200 dark-mode:bg-gray-800">
+            <div
+              x-data="{ open: true }"
+              class="flex flex-col max-w-screen-xl px-4 mx-auto md:items-center md:justify-between md:flex-row md:px-6 lg:px-8 bg-[#292937] w-[300vh]"
+            >
+              <div class="flex items-center justify-between p-4 ">
+                <img src={Logo} alt="" />
```
Comment (akbarali0705) [inline src/components/navbar/Navbar.jsx]: _**Logoda cursor pointer bo'lishi kk**_

File: src/components/navbar/Navbar.jsx [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+                    className="w-6 h-6 text-white"
+                  >
+                    <path
+                      x-show="!open"
+                      fill-rule="evenodd"
+                      d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM9 15a1 1 0 011-1h6a1 1 0 110 2h-6a1 1 0 01-1-1z"
+                      clip-rule="evenodd"
+                    ></path>
+                  </svg>
+                </button>
```
Comment (tolibjon1015) [inline src/components/navbar/Navbar.jsx:29]: nimaga buttonni ichida svg ????????????

**Review by tolibjon1015 — CHANGES_REQUESTED** [review]

**Review by Muhammadali2oo6 — CHANGES_REQUESTED** [review]
mistakes kop

**Review by akbarali0705 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**